lib/routes/twitter/utils.ts

```diff
@@ -495,4 +496,15 @@ export const excludeRetweet = function (tweets) {
     return excluded;
 };
 
-export default { ProcessFeed, getAppClient, parseRouteParams, excludeRetweet };
+export const keepOnlyMedia = function (tweets) {
+    const excluded = [];
+    for (const t of tweets) {
+        if (!t.extended_entities || !t.extended_entities.media) {
+            continue;
+        }
+        excluded.push(t);
+    }
+    return excluded;
```

Comment:
This can be simplified into
```suggestion
    const excluded = tweets.filter((t) => t.extended_entities && t.extended_entities.media);
    return excluded;
```